node/node_newblock.go

```diff
@@ -156,14 +160,20 @@ func (node *Node) proposeNewBlock() (*types.Block, error) {
 
 	if isBeaconchainInCrossLinkEra {
 		allPending, err := node.Blockchain().ReadPendingCrossLinks()
-
+		invalidToDelete := []types.CrossLink{}
 		if err == nil {
 			for _, pending := range allPending {
-				if err = node.VerifyCrossLink(pending); err != nil {
-					continue
-				}
 				exist, err := node.Blockchain().ReadCrossLink(pending.ShardID(), pending.BlockNum())
 				if err == nil || exist != nil {
+					invalidToDelete = append(invalidToDelete, pending)
```

Comment:
do we have some number on how many invalid cross links we are seeing from the log?